database-management/migrations/20180402170153_make-varchar-uniform.js

```diff
@@ -0,0 +1,156 @@
+/* eslint-disable */
```

Comment:
The async/await bug is gone with the new eslint version and I removed all the `eslint-disables` from all the other migrations so hopefully you can just remove it and let prettier do it's thing and it won't introduce any new errors